src/lunamary/services/CommonService.java

```diff
@@ -0,0 +1,90 @@
+package lunamary.services;
+
+import datastructures.circulardoublylinkedlist.MyCircularDoublyLinkedList;
+import lunamary.modelPerson.Parent;
+import lunamary.modelPerson.Student;
+import lunamary.modelPerson.Teacher;
+import lunamary.modelSchool.Classroom;
+import lunamary.modelSchool.School;
+import lunamary.modelSchool.Subject;
+
+public class CommonService {
+
+
+    public CommonService() {
+
+    }
+
+    public void print(String text) {
```

Comment:
Not sure why are you mixing these prints with service data but is not ok to do that.